Integreat/Integreat.Shared/Data/Loader/Targets/EventPagesDataLoader.cs

```diff
@@ -10,16 +10,24 @@ namespace Integreat.Shared.Data.Loader.Targets
     /// <inheritdoc />
     public class EventPagesDataLoader : IDataLoader
     {
-        public const string FileNameConst = "eventsV1";
-
-        /// <inheritdoc />
-        public string FileName => FileNameConst;
+        private const string _FileNameConst = "eventsV3";
+        private string _FileName;
 
         public DateTime LastUpdated
         {
             get => Preferences.LastPageUpdateTime<EventPage>(_lastLoadedLanguage, _lastLoadedLocation);
             // ReSharper disable once ValueParameterNotUsed
             set => Preferences.SetLastPageUpdateTime<EventPage>(_lastLoadedLanguage, _lastLoadedLocation, DateTime.Now);
+        }
+
+        public string FileName
```

Comment:
change also to auto-property  here

and should we extract a base class here? it is the same like disclaimer...